test/wdio/wdio-globals.spec.ts

```diff
@@ -0,0 +1,38 @@
+import { browser } from '@wdio/globals';
+import AxeBuilder from '@axe-core/webdriverio';
+import { assert } from 'chai';
+import express from 'express';
+import { fixturesPath } from 'axe-test-fixtures';
+import { Server, createServer } from 'http';
+import testListen from 'test-listen';
+
+describe('@wdio/globals', () => {
+  let server: Server;
+  let addr: string;
+  let builder: AxeBuilder;
+  before(async () => {
+    const app = express();
+    app.use(express.static(fixturesPath));
+    server = createServer(app);
+    addr = await testListen(server);
```

Comment:
I know we use `testListen` in the other packages for their tests, it appears it's now deprecated and moved over to [`aync-listen`](https://github.com/vercel/async-listen). We should update it here and create a tech-debt to update it in all the other packages?

Reply:
I think that work should be done completely in a different pr.

Reply:
https://github.com/dequelabs/axe-core-npm/issues/884